Extensions/tweaks.js

```diff
@@ -786,11 +792,11 @@ XKit.extensions.tweaks = new Object({
 				${notificationBadgeStyle}
 			}`, 'xkit_tweaks_notification_badge_style');
 		}
-		
+
```

Comment:
whitespace changes were picked up by Atom applying the repo's `.editorconfig`, fyi